src/api/routes/docs/plugin_docs.rs

```diff
@@ -0,0 +1,74 @@
+use crate::{models::ApiResponse, services::plugins::PluginCallResponse};
+
+/// Calls a plugin method.
+#[utoipa::path(
+    get,
```

Comment:
This should be post

Reply:
my bad, in the sdk I have modified openapi manually